web/src/constants.ts

```diff
@@ -38,4 +38,21 @@ export const DEFAULT_MINISTRY = 'CITZ';
 
 export const LAYOUT_SET_MIN = 'min';
 export const LAYOUT_SET_AUTH = 'auth';
-export const LAYOUT_SET_UNAUTH = 'unauth';
\ No newline at end of file
+export const LAYOUT_SET_UNAUTH = 'unauth';
+
+export const COMPONENT_METADATA = [
+  { displayName: 'Notification: Email', inputValue: 'notificationEmail' },
+  { displayName: 'Notification: SMS', inputValue: 'notificationSMS' },
+  { displayName: 'Notification: MS Teams', inputValue: 'notificationMSTeams' },
+  { displayName: 'Payment processing: Bambora', inputValue: 'paymentBambora' },
+  { displayName: 'Payment processing: PayBC', inputValue: 'paymentPayBC' },
+  { displayName: 'File Transfer', inputValue: 'fileTransfer' },
+  { displayName: 'File Storage', inputValue: 'fileStorage' },
+  { displayName: 'Geo Mapping: Web-based', inputValue: 'geoMappingWeb' },
+  { displayName: 'Geo Mapping: Location Services', inputValue: 'geoMappingLocation' },
+  { displayName: 'Scheduling: Calendar', inputValue: 'schedulingCalendar' },
+  { displayName: 'Scheduling: Appointments', inputValue: 'schedulingAppointments' },
+  { displayName: 'Identity Management: SiteMinder', inputValue: 'identityManagementSiteMinder' },
```

Comment:
Just a note, in my last commit `identityManagement` was changed to `idm` as per my PR review. So `identityManagementSiteMinder` will be `idmSiteMinder`. This is carries for `idmKeyCloak` and `idmActiveDir`

This can be seen in the [Profile controller](https://github.com/bcgov/platform-services-registry/blob/featuer/issue-76/api/src/controllers/profile.ts#L146-L148)